markless.el

```diff
@@ -355,7 +355,30 @@ Marks PRE and POST as markup and the content with PROP."
                              nil nil nil nil
                              (font-lock-multiline . t)))
   (setq-local flyspell-generic-check-word-predicate
-              #'markless-at-word-p))
+              #'markless-at-word-p)
+  (setq-local comment-start ";")
```

Comment:
You should probably also set `comment-start-skip` to something like `";+ "` (note the space).